tests/test_correct_year.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+    test_correct_year
```

Comment:
I would say that this test is not about the gettext builder, but the feature that corrects copyright years from the SOURCE_DATE_EPOCH environment variable value. This feature concerns all builders.